playground/env/index.html

```diff
@@ -14,6 +14,7 @@ <h1>Environment Variables</h1>
 <p>import.meta.env.VITE_INLINE: <code class="inline"></code></p>
 <p>typeof import.meta.env.VITE_BOOL: <code class="bool"></code></p>
 <p>process.env.NODE_ENV: <code class="node-env"></code></p>
+<p>process.env.VITE_EXPAND: <code class="expand"></code></p>
```

Comment:
I think this needs to align with line 33 below
```suggestion
<p>import.meta.env.VITE_EXPAND: <code class="expand"></code></p>
```

Reply:
Yes, thank you for catch it